tests/integration/questionnaire/test_questionnaire_interstitial.py

```diff
@@ -24,7 +24,7 @@ def test_interstitial_page_button_text_is_continue(self):
         resp_url, resp = self.postRedirectGet(resp_url, post_data)
         content = resp.get_data(True)
 
-        self.assertRegex("/Continue$", content)
+        self.assertTrue("Continue" in content)
```

Comment:
The prior gives a better error if this fails otherwise we'll get expected true was false not "expecting … to contain Continue"